pkg/chartrenderer/default.go

```diff
@@ -79,10 +82,11 @@ func DiscoverCapabilities(disc discovery.DiscoveryInterface) (*chartutil.Capabil
 
 // Render loads the chart from the given location <chartPath> and calls the Render() function
 // to convert it into a ChartRelease object.
+// Deprecated: Use RenderEmbeddedFS for new code!
 func (r *chartRenderer) Render(chartPath, releaseName, namespace string, values interface{}) (*RenderedChart, error) {
```

Comment:
This function is used by extensions also? We probably should not deprecate it as it is not sure whether the other repo will switch to `embed` or not.

Reply:
Since there is no as say deadline that after this many releases or months this function will be removed probably we should not deprecate it. Not very important though.

Reply:
I have extended the release note with

> They will be removed in a future version. You should consider adapting your code to the newly introduced methods.

to make it more clear.